heron/spi/src/java/com/twitter/heron/spi/common/defaults.yaml

```diff
@@ -87,11 +87,12 @@ SANDBOX_OVERRIDE_YAML:        ${HERON_SANDBOX_CONF}/override.yaml
 ################################################################################
 # Default values for heron sandbox binaries and jars
 ################################################################################
-SANDBOX_EXECUTOR_BINARY:      ${HERON_SANDBOX_BIN}/heron-executor
-SANDBOX_STMGR_BINARY:         ${HERON_SANDBOX_BIN}/heron-stmgr
-SANDBOX_TMASTER_BINARY:       ${HERON_SANDBOX_BIN}/heron-tmaster
-SANDBOX_SHELL_BINARY:         ${HERON_SANDBOX_BIN}/heron-shell
-SANDBOX_SCHEDULER_JAR:        ${HERON_SANDBOX_LIB}/scheduler/heron-scheduler.jar
+SANDBOX_EXECUTOR_BINARY:          ${HERON_SANDBOX_BIN}/heron-executor
+SANDBOX_STMGR_BINARY:             ${HERON_SANDBOX_BIN}/heron-stmgr
+SANDBOX_TMASTER_BINARY:           ${HERON_SANDBOX_BIN}/heron-tmaster
+SANDBOX_SHELL_BINARY:             ${HERON_SANDBOX_BIN}/heron-shell
+SANDBOX_PYHERON_INSTANCE_BINARY:  ${HERON_SANDBOX_BIN}/pyheron_st_instance
```

Comment:
why call pyheron_st_instance? simply calling heron-python - will make it easier to remember and also  sticks to the convention?